data/json/harvest.json

```diff
@@ -354,7 +354,8 @@
       { "drop": "meat_tainted", "type": "flesh", "mass_ratio": 0.25 },
       { "drop": "fat_tainted", "type": "flesh", "mass_ratio": 0.08 },
       { "drop": "bone_tainted", "type": "bone", "mass_ratio": 0.1 },
-      { "drop": "raw_tainted_leather", "type": "skin", "mass_ratio": 0.02 }
+      { "drop": "raw_tainted_leather", "type": "skin", "mass_ratio": 0.02 },
+      { "drop": "pheromone", "type": "bionic", "base_num": [ 0, 1 ], "scale_num": [ 0.3, 0.6 ] }
```

Comment:
Mimicking pheromone as a bionic feels... wrong.

Reply:
JSON_INFO.md states that "bionic" entry in harvest.json doesn't necessarily have to be a CBM. It generally has to be "something" that you extract via dissecting. It was introduced recently when this part was jsonized, but because there was nothing else then CBMs that would fall under dissection, the names stayed. I may, if you wish, rename that entry to "dissection" or something, but that would require adding extra migration code and renaming functions in activity handlers. Doable but tedious. Should I do it anyway?

Reply:
No, if documentation says it's ok, then it's ok.